### ISSUE TITLE
Fall damage disabled when you spam the BOOST effect

### ISSUE THREAD

**Issue by Amph1dian** [GC-spigot/AdvancedEnchantments#2701]
### Describe the bug

When you spam the BOOST effect fall damage for the player will get disabled. the only way i found to fix this was to restart the server.

### How to reproduce

1. copy and paste this enchant into the enchants.yml file: https://pastebin.com/zxwajQQ1
2. go in game and do /ae reload
3. do /minecraft:tp ~ ~20 ~ and see if you take damage
4. grab a sword and do /ae enchant boostbug 1
5. hold the sword in your hand and spam LMB for a second or two
6. once you land on the ground you shouldn't take any fall damage.
7. do /minecraft:tp ~ ~20 ~ and you shouldn't take fall damage.

### Screenshots / Videos

https://www.youtube.com/watch?v=VMeFGxWE3Vc

### "/ae plinfo" link

https://paste.md-5.net/uviwohacac

**Comment by Burchard36** [comment]
Tested this myself and was able to replicate

+1 for providing the enchantment and the exact steps to reproduce this :)

Will work on a fix for this in an upcoming update!

**Comment by Burchard36** [comment]
Hello there, 8.9.11 update will fix this issue, thanks for the report!

**Comment by Burchard36** [comment]
8.9.11 is out now! Please update to the latest version :)

If you have any new issues please open a new issue request!
